gdal/scripts/detect_tabulations.sh

```diff
@@ -19,7 +19,7 @@ cd "$GDAL_ROOT"
 ret_code=0
 
 echo "Checking for tabulation characters..."
-if find alg gnm port ogr gcore frmts apps \( -name "*.cpp" -o -name "*.h" \) -exec sh -c "grep -P '\\t' {} >/dev/null && echo {}" \; | grep -v frmts/msg/PublicDecompWT | grep -v frmts/jpeg/libjpeg | grep -v frmts/gtiff/libtiff | grep -v frmts/gtiff/libgeotiff | grep -v frmts/grib/degrib | grep -v ogr/ogrsf_frmts/geojson/libjson | grep -v frmts/hdf4/hdf-eos | grep -v frmts/gif/giflib | grep -v frmts/pcraster/libcsf | grep -v frmts/png/libpng | grep -v cpl_mem_cache ; then
+if grep -r --files-with-matches -P --include='alg/*.h alg/*.cpp gnm/*.h gnm/*.cpp port/*.h port/*.cpp ogr/*.h ogr/*.cpp gcore/*.h gcore/*.cpp frmts/*.h frmts/*.cpp apps/*.h apps/*.cpp' '\\t' | grep -v -e frmts/msg/PublicDecompWT -e frmts/jpeg/libjpeg -e frmts/gtiff/libtiff -e frmts/gtiff/libgeotiff -e frmts/grib/degrib -e ogr/ogrsf_frmts/geojson/libjson -e frmts/hdf4/hdf-eos -e frmts/gif/giflib -e frmts/pcraster/libcsf -e frmts/png/libpng -e cpl_mem_cache ; then
```

Comment:
No idea what's wrong in that change, but in practice it doesn't scan any file...